vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/SubmitWorkflowRequest.java

```diff
@@ -43,6 +43,19 @@ public boolean equals(Object o) {
         && Objects.equals(workflowVersion, that.workflowVersion);
   }
 
+  public boolean equalsIgnoreAttempt(SubmitWorkflowRequest that) {
```

Comment:
Could equals() now return `equalsIgnoreAttempt(that) && attempt == that.attempt`?

Reply:
Yes, if you prefer. I didn't because it's one of the IDE-generated methods.

Reply:
If you need to add another field it means 1 place to update instead of 2, and the second one might get missed?

Reply:
Sounds good. I've done the same for `hashCode`.